python/gsm/receiver/multiarfcns_receiver.py

```diff
@@ -0,0 +1,198 @@
+#!/usr/bin/env python2
```

Comment:
Either change this to python3 or remove the whole shebang as this is not really an executable.

Reply:
Solved.